### DISCUSSION TITLE
Videos para HTML y CSS

### DISCUSSION THREAD

**Discussion by SujeiRodriguez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#169] [Show and tell]
Buenas noches, quería compartirles unos videos, por si quieren mas información sobre los temas y muchos otros mas, hay varios temas desde cero, así que creo que les puede servir.

Buscar en YOUTUBE,  "**píldoras informáticas html**" o css, etc...

Me parece que explica muy bien y si son un poco mas extensos los videos.

A lo mejor ya lo conocían, pero si no, espero les sirva.

**Comment by JuanSalvador73** [comment]
En mi caso si ya los conocía, pero siguen siendo muy buenos. Gracias por la recomendación y el recordatorio.
